iina/PlaylistViewController.swift

```diff
@@ -307,16 +307,17 @@ class PlaylistViewController: NSViewController, NSTableViewDataSource, NSTableVi
   // MARK: - IBActions
 
   @IBAction func addToPlaylistBtnAction(_ sender: AnyObject) {
-    Utility.quickOpenPanel(title: "Add to playlist", isDir: false) { (url) in
-      if url.isFileURL {
-        self.playerCore.addToPlaylist(url.path)
-        self.reloadData(playlist: true, chapters: false)
-        self.mainWindow.displayOSD(.addToPlaylist(1))
+    Utility.quickMultipleOpenPanel(title: "Add to playlist") { urls in
+      for url in urls {
+        if url.isFileURL {
+          self.playerCore.addToPlaylist(url.path)
+          self.mainWindow.displayOSD(.addToPlaylist(1))
```

Comment:
I see. I had trouble understanding what the `displayOSD` is for and the meaning of the `Int` parameter in the `addToPlaylist` enum.